### ISSUE TITLE
Mucus in stool

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9946]
**Preferred term label:**
Mucus in stool

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Anisakiasis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000686
- label: Mucus in stool
- parents: [Abnormal stool composition (HP:0031685)](https://hpo.jax.org/app/browse/term/HP:0031685)
- pmid: 
- def: Abnormally increased amount of mucus in the feces. Mucus is a slippery aqueous secretion produced by mucous membranes including those of the intestines.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9946
- synonyms:
